microprofile.jdt/org.eclipse.lsp4mp.jdt.tp/org.eclipse.lsp4mp.jdt.target

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="Running Platform">
+	<locations>
+		<location type="Target" uri="https://raw.githubusercontent.com/eclipse/eclipse.jdt.ls/master/org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target" />
+		<location includeSource="true" missingManifest="generate" type="Maven">
+			<feature id="io.smallrye.common.expression" label="Smallrye Commons Expression" provider-name="Smallrye" version="1.0.0.qualifier" />
+			<dependencies>
+				<dependency>
+					<groupId>io.smallrye.common</groupId>
+					<artifactId>smallrye-common-expression</artifactId>
+					<version>1.12.0</version>
+				</dependency>
+			</dependencies>
+		</location>
+		<location includeSource="true" missingManifest="generate" type="Maven">
+			<feature id="io.smallrye.common.function" label="Smallrye Commons Function" provider-name="Smallrye" version="1.0.0.qualifier" />
+			<dependencies>
+				<dependency>
+					<groupId>io.smallrye.common</groupId>
+					<artifactId>smallrye-common-function</artifactId>
+					<version>1.12.0</version>
+				</dependency>
+			</dependencies>
+		</location>
+		<location includeSource="true" missingManifest="generate" type="Maven">
+			<feature id="io.smallrye.common.constraint" label="Smallrye Commons Constraint" provider-name="Smallrye" version="1.0.0.qualifier" />
+			<dependencies>
+				<dependency>
+					<groupId>io.smallrye.common</groupId>
+					<artifactId>smallrye-common-constraint</artifactId>
+					<version>1.12.0</version>
+				</dependency>
+			</dependencies>
+		</location>
+		<location includeSource="true" missingManifest="generate" type="Maven">
+			<feature id="org.jboss.logging" label="JBoss Logging" provider-name="JBoss" version="1.0.0.qualifier" />
```

Comment:
It looks like I could also reference a slightly out of date copy of the library from orbit. Would that be better?